frontend/src/_data/content/index.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-title: ReportStream makes it really easier to share public health data
```

Comment:
Test this other thing

Reply:
I made the update you asked for

Reply:
I'm still not sure if bold is better than all caps

Reply:
verbally chatted on 3/23, drew is cool with bold